internal/app/machined/internal/phase/phase.go

```diff
@@ -52,22 +52,27 @@ func NewRunner(config config.Configurator) (*Runner, error) {
 		return nil, err
 	}
 
-	mode := platform.Mode()
-	switch mode {
-	case runtime.Metal:
-		fallthrough
-	case runtime.Cloud:
+	var mode runtime.Mode
+
+	switch platform.Name() {
+	case "Container":
```

Comment:
Trying to find a better way to handle this.

Reply:
Why move it from `platform.Mode()`?

Reply:
Import cycle issue. Still chewing on this to find a better way. But the issue is that `runtime` imports platform, and platform imports `runtime`.